docker/docker_test.go

```diff
@@ -15,7 +15,10 @@ func TestMain(m *testing.M) {
 	}
 	exitVal := m.Run()
 
-	teardownSubtest()
+	err := teardownSubtest()
+	if err != nil {
+		fmt.Printf("welp teardown went terribly...not sure if it matters or not but here is your error: %v", err)
```

Comment:
Probs could use `t.Error()` if you want the test to fail.

Reply:
This is another one of those weird tests :) doing setup and teardown i use TestMain since it gives me more control. So there is no `t.Error()`......

https://medium.com/goingogo/why-use-testmain-for-testing-in-go-dafb52b406bc

So inside i did more `os.Exit` and did a non zero exit code.